tests/unit/test_published_clusters_with_data.py

```diff
@@ -0,0 +1,51 @@
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+
+
+@responses.activate
+def test_published_clusters_with_data():
+    project_config = {
+        "name": "Project 1",
+        "description": "Mastering Project",
+        "type": "DEDUP",
+        "unifiedDatasetName": "Project 1 - Unified Dataset",
+        "externalId": "Project1",
+        "resourceId": "1",
+    }
+
+    unified_dataset_json = {
+        "id": "unify://unified-data/v1/datasets/8",
+        "name": "Project_1_unified_dataset",
+        "version": "10",
+        "relativeId": "datasets/8",
+        "externalId": "Project_1_unified_dataset",
+    }
+
+    rcwd_json = {
+        "externalId": "1",
+        "id": "unify://unified-data/v1/datasets/36",
+        "name": "Project_1_unified_dataset_dedup_published_clusters_with_data",
```

Comment:
The name of this dataset needs to align with that of the unified dataset (it currently does) and the unified dataset name provided in the project (line 13, it currently does not)

Reply:
most of this file is just a copy of test_record_clusters_with_data , so this error might be present in that file too-- should I change that as well?

Reply:
Not as part of this PR, no.